lib/modules/apostrophe-areas/lib/api.js

```diff
@@ -350,10 +350,16 @@ module.exports = function(self, options) {
 
   self.getSchemaOptions = function(doc, name) {
     if (!(doc && doc.type)) {
-      throw new Error('You did not pass a doc as the first argument to getSchemaOptions, which expects (doc, name)');
+      // Probably an area or singleton helper call without a
```

Comment:
If this is still a mistake (even if not a fatal one) then we should still log a warning.

Reply:
It is not any kind of mistake. It's an apos.area call with an options object rather than (doc, name), which is less common but entirely valid.

Reply:
The mistake lay in me not realizing it would be called in this situation and making it too picky.